hq/test/logic/VulnerableAccessKeysTest.scala

```diff
@@ -0,0 +1,80 @@
+package logic
+
+import logic.VulnerableAccessKeys._
+import model.{AccessKey, AccessKeyDisabled, AccessKeyEnabled}
+import org.joda.time.DateTime
+import org.scalatest.{FreeSpec, Matchers}
+
+
+class VulnerableAccessKeysTest extends FreeSpec with Matchers {
+  def accessKey(daysOld: Int, active: Boolean) = {
+    AccessKey(
+      if (active) AccessKeyEnabled else AccessKeyDisabled,
+      Some(DateTime.now().minusDays(daysOld))
+    )
+  }
+
+  "hasOutdatedHumanKey" - {
```

Comment:
I wonder if the `hasOutdatedHumanKey` and the `hasOutdatedMachineKey` tests are required? I couldn't see where these methods were being used.

Reply:
For sure, if the functions themselves aren't used we can remove these at the same time 👍